### PR TITLE
Get ready for initial internal release

### PR DESCRIPTION
**Pull request by yuft**
This PR finished the bridge work.

It builds all SDKs. 

GHA publishes NPM package to Displayr NPM repo.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+          path: ${{ github.workspace}}/sdk/
+      - name: Uncompress nodejs SDK
+        run: tar -zxf ${{github.workspace}}/sdk/nodejs.tar.gz -C ${{github.workspace}}/sdk/nodejs
+      - run: python -m pip install pip twine
+      - env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+        name: Publish SDKs
+        run: |
+
+          # DY Hack, update nodejs package.json name to be @displayr/pulumi-signalfx-synthetics
```
Comment (yuft) [inline .github/workflows/release.yml:257]: hack so that we can publish.

File: provider/resources.go [hunk 2]
```diff
@@ -123,51 +123,51 @@ func Provider() tfbridge.ProviderInfo {
 		// PluginDownloadURL is an optional URL used to download the Provider
 		// for use in Pulumi programs
 		// e.g https://github.com/org/pulumi-provider-name/releases/
-		PluginDownloadURL: "",
+		PluginDownloadURL: "https://github.com/Displayr/pulumi-signalfx-synthetics/releases/download/v0.0.12/",
```
Comment (yuft) [inline provider/resources.go:126]: specify the version here to work with Pulumi's custom publish script https://github.com/jaxxstorm/scripts/tree/third_party/ci

File: sdk/dotnet/Provider.cs [hunk 3]
```diff
@@ -55,6 +55,7 @@ private static CustomResourceOptions MakeResourceOptions(CustomResourceOptions?
             var defaultOptions = new CustomResourceOptions
             {
                 Version = Utilities.Version,
+                PluginDownloadURL = "https://github.com/Displayr/pulumi-signalfx-synthetics/releases/download/v0.0.11/",
```
Comment (jankurianski) [inline sdk/dotnet/Provider.cs]: Should be updated/regenerated to point to 0.0.12?

File: sdk/go/synthetics/pulumi-plugin.json [hunk 4]
```diff
@@ -1,4 +1,5 @@
 {
   "resource": true,
-  "name": "synthetics"
+  "name": "synthetics",
+  "server": "https://github.com/Displayr/pulumi-signalfx-synthetics/releases/download/v0.0.11/"
```
Comment (jankurianski) [inline sdk/go/synthetics/pulumi-plugin.json]: Ditto

**Review by jankurianski — APPROVED** [review]
lgtm

File: sdk/go/synthetics/pulumi-plugin.json (hunk 4 above)
Comment (yuft) [reply sdk/go/synthetics/pulumi-plugin.json]: yep, the pipeline re-generate everything under /sdk folder.

I will re-run `make build_sdks` locally.

**Pull request merged**